client/src/components/Questions/QuestionsList/Question/Question.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
```

Comment:
It is great that you layout you components with class. It is easier for us to work on.